kcwarden/cli.py

```diff
@@ -107,6 +107,12 @@ def add_audit_parser(subparsers):
         action="store_true",
     )
 
+    parser_audit.add_argument(
+        "--fail-on-findings",
```

Comment:
Do we have docs for the command line parameters? If yes, this should be added.

Reply:
Good catch. Added with f243dba.